### PR TITLE
List view improvement

### PR DESCRIPTION
**Pull request by Rajat-Dabade**
Loom: https://www.loom.com/share/ec77eac228fb4d35a4025fa16a8f5285?sid=bd27b1a0-771a-4be4-b0ce-c8e339a290bb

Fixes: 
https://github.com/SigNoz/signoz/issues/4572
https://github.com/SigNoz/signoz/issues/4574

### CODE REVIEW

File: frontend/src/container/NewWidget/LeftContainer/index.tsx [hunk 1]
```diff
@@ -22,6 +23,7 @@ function LeftContainer({
 }: WidgetGraphProps): JSX.Element {
 	return (
 		<>
+			{/* {selectedGraph !== PANEL_TYPES.LIST && ( */}
```
Comment (YounixM) [inline frontend/src/container/NewWidget/LeftContainer/index.tsx:26]: Remove commented code.

File: frontend/src/container/NewWidget/LeftContainer/index.tsx [hunk 2]
```diff
@@ -6,6 +6,7 @@ import ExplorerColumnsRenderer from './ExplorerColumnsRenderer';
 import QuerySection from './QuerySection';
 import { QueryContainer } from './styles';
 import WidgetGraph from './WidgetGraph';
+// import WidgetListView from './WidgetListView/WidgetListView';
```
Comment (YounixM) [inline frontend/src/container/NewWidget/LeftContainer/index.tsx:9]: Remove commented code.

File: frontend/src/container/NewWidget/LeftContainer/WidgetGraph/index.tsx [hunk 3]
```diff
@@ -41,6 +42,9 @@ function WidgetGraph({
 			selectedTime: selectedTime.enum,
 		},
 		selectedDashboard?.data?.version || DEFAULT_ENTITY_VERSION,
+		{
```
Comment (YounixM) [inline frontend/src/container/NewWidget/LeftContainer/WidgetGraph/index.tsx:45]: We don't need a query-range API call for list panels?

File: frontend/src/container/NewWidget/LeftContainer/index.tsx (hunk 1 above)
Comment (Rajat-Dabade) [reply frontend/src/container/NewWidget/LeftContainer/index.tsx]: Removed.

File: frontend/src/container/NewWidget/LeftContainer/index.tsx (hunk 2 above)
Comment (Rajat-Dabade) [reply frontend/src/container/NewWidget/LeftContainer/index.tsx:9]: Removed.

File: frontend/src/container/NewWidget/LeftContainer/WidgetGraph/index.tsx (hunk 3 above)
Comment (YounixM) [reply frontend/src/container/NewWidget/LeftContainer/WidgetGraph/index.tsx:45]: Why separate handling for List view only?

Comment (Rajat-Dabade) [reply frontend/src/container/NewWidget/LeftContainer/WidgetGraph/index.tsx:45]: So we have a navigation, to fetch the next and previous data from inside the panel, also data fetching is based on pagination, which other panels do not have. Handling it from the parent will be drilling down too many props for the child. That's the reason it was handled separately.

**Comment by YounixM** [comment]
Related to https://github.com/SigNoz/signoz/pull/4675

**Pull request closed without merging**